.github/workflows/developing.yml

```diff
@@ -35,6 +31,12 @@ jobs:
           # Source https://github.com/expo/expo-github-action/issues/20#issuecomment-541676895
           echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
 
+      - name: Checkout (content)
+        uses: actions/checkout@v4
+        with:
+          repository: mdn/content
+          path: mdn/content
```

Comment:
Checking content out later prevents `yarn [install]` in the root folder above from running `yarn install` in this folder.